classes/Twig/Block/UpgradeButtonBlock.php

```diff
@@ -179,6 +179,7 @@ public function render()
             'downloadPath' => $this->downloadPath . DIRECTORY_SEPARATOR,
             'directoryVersionNumber' => $this->config->get('directory.version_num'),
             'manualMode' => $this->manualMode,
+            'phpVersion' => phpversion(),
```

Comment:
To PHP devs, could you say me if the use of phpversion() is better/slower/the same as the defined PHP_VERSION ? Just to know, not a big deal, of course :)

Reply:
It's always faster to use predefined constant instead of calling a function.
```suggestion
            'phpVersion' => PHP_VERSION,
```